support-frontend/assets/components/paymentMethodSelector/PaymentMethodSelectorContainer.tsx

```diff
@@ -0,0 +1,64 @@
+import { contributionTypeIsRecurring } from 'helpers/contributions';
+import { getValidPaymentMethods } from 'helpers/forms/checkouts';
+import { getFullExistingPaymentMethods } from 'helpers/forms/existingPaymentMethods/existingPaymentMethods';
+import type { PaymentMethod } from 'helpers/forms/paymentMethods';
+import { isContribution } from 'helpers/redux/checkout/product/selectors/productType';
+import { useContributionsSelector } from 'helpers/redux/storeHooks';
+import type { PaymentMethodSelectorProps } from './paymentMethodSelector';
+
+function PaymentMethodSelectorContainer({
+	render,
+}: {
+	render: (
+		paymentMethodSelectorProps: PaymentMethodSelectorProps,
+	) => JSX.Element;
+}): JSX.Element {
+	const contributionType = useContributionsSelector(
+		(state) => state.page.checkoutForm.product.productType,
+	);
```

Comment:
```suggestion
	const contributionType = useContributionsSelector(getContributionType);
```
The `getContributionType` selector guarantees that you'll receive a `ContributionType`, so no need to use an additional `isContribution` check later.